src/components/App/App.module.css

```diff
@@ -1,8 +1,24 @@
+body {
+	background-color: #FCE4EC;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#FCE4EC" to be "#fce4ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/TODO-Olga/pullRequest?prid=5555083)